network/update-proxy-configs

```diff
@@ -119,21 +119,26 @@ fi
 # Pacman (archlinux) also
 if [ -d /etc/pacman.d ]; then
     if [ -n "$PROXY_ADDR" ]; then
-        cat > /etc/pacman.d/01-qubes-proxy.conf <<EOF
+        mkdir -p /run/qubes/bin
+        cat > /run/qubes/bin/pacman <<EOF
+#!/bin/bash
 ### This file is automatically generated by Qubes ($0 script).
 ### All modifications here will be lost.
-### If you want to override some of this settings, create another file under 
-### /etc/pacman.d
-
-[options]
-# Use Qubes Update Proxy
-XferCommand = /usr/bin/env ALL_PROXY=$PROXY_ADDR /usr/bin/curl -C - -f -o %o %u
+exec env ALL_PROXY=$PROXY_ADDR /usr/bin/pacman \$@
```

Comment:
ah, one more thing: it should be `"$@"`, otherwise arguments with spaces won't be handled correctly.

Reply:
Can you give me an example of such command so next time I can do a better testing? I tried with ```pacman -S -y -u --color always``` and it worked without issues.

Reply:
It's about arguments with spaces inside, for example:
`pacman --print --print-format "package %n version %v" -U`
(not sure if I got the syntax right, written based on pacman man page)

Reply:
I tried your command and it failed without the double quotes, thanks.